recorder/pom.xml

```diff
@@ -135,7 +135,7 @@
         <profile>
             <id>native</id>
             <activation>
-                <activeByDefault>true</activeByDefault>
+                <activeByDefault>false</activeByDefault>
```

Comment:
I had not setup recorder in this repo (had set it up on mac in your fork of honest-profiler). Also, I figured since there are OSX-specific changes in cmake, there is a need to identify all changes made on my machine and add them to cmake (for osx) in this repo. To avoid build failures, I changes this to "false" but incorectly made this part of the PR